docs/API.md

```diff
@@ -1226,7 +1225,7 @@ An object with the following properties:
 
 ### API method: `user_create`
 
-In order to use the [`add_batch_job`][add_batch_job] method a
+In order to use the [`batch_create`][batch_create] method a
 [*username*][Username] and its [*api key*][Api key] must be added by this method.
 
 This method is not available if [`RPCAPI.enable_add_api_user`][RPCAPI.enable_add_api_user]
```

Comment:
Should not this be changed to the new name?

On line 1344:
```
    "message": "Call to \"add_api_user\" method not permitted from a remote IP"                                                                                   
```

On line 1354:
```
    "message": "Procedure 'add_api_user' not found"                                                                                                               
```

On lines 1367-1369:
```
A [*username*][Username] and its [*api key*][Api key] can be added with the
[`add_api_user`][add_api_user] method. A [*username*][Username] can only have
one un-finished *batch* at a time.
```

Reply:
good catch, I've fixed it